src/cluster/cluster_defs.h

```diff
@@ -41,6 +41,7 @@ inline constexpr const char *errNoMasterNode = "The node isn't a master";
 inline constexpr const char *errClusterNoInitialized = "The cluster is not initialized";
 inline constexpr const char *errInvalidClusterNodeInfo = "Invalid cluster nodes info";
 inline constexpr const char *errInvalidImportState = "Invalid import state";
+inline constexpr const char *errYouAreFired = "You are fired";
```

Comment:
I think it's not a right wording. Could be something like `disconnected due to the topology change` or others.